packages/web-runtime/src/services/auth/userManager.ts

```diff
@@ -284,6 +286,52 @@ export class UserManager extends OidcUserManager {
     this.store.commit('SET_CAPABILITIES', capabilities)
   }
 
+  // copied from upstream oidc-client-ts UserManager with CERN customization
+  protected async _signinEnd(url: string, verifySub?: string): Promise<User> {
```

Comment:
maybe we should use `...args` instead to be more resilient against api changes

Reply:
... or rather: additionally

Reply:
good idea